sycl/source/detail/config.cpp

```diff
@@ -175,8 +176,21 @@ getSyclDeviceTypeMap() {
   return SyclDeviceTypeMap;
 }
 
-// Array is used by SYCL_DEVICE_FILTER and SYCL_DEVICE_ALLOWLIST and
-// ONEAPI_DEVICE_SELECTOR
+#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
+// Array is used by SYCL_DEVICE_ALLOWLIST and ONEAPI_DEVICE_SELECTOR.
+const std::array<std::pair<std::string, backend>, 7> &getSyclBeMap() {
+  static const std::array<std::pair<std::string, backend>, 7> SyclBeMap = {
+      {{"host", backend::host},
```

Comment:
Is comment on line 165 applicable here as well?

Reply:
Yes. I'll add the comment.